ffv1.md

```diff
@@ -2,73 +2,85 @@
 
-This document defines version 0, 1 and 3 of FFV1. The distinctions of the versions are provided throughout the document, but in summary:{V3}
+This document defines versions 0, 1, and 3 of FFV1. The distinctions of the versions are provided throughout the document, but in summary:{V3}
 
-- Version 0 of FFV1 was the original implementation of FFV1 and has been flagged as stable on April 14, 2006 [@?FFV1_V0].{V3}
+- Version 0 of FFV1 was the original implementation of FFV1 and was flagged as stable on April 14, 2006 [@?FFV1_V0].{V3}
 
-- Version 1 of FFV1 adds support of more video bit depths and has been has been flagged as stable on April 24, 2009 [@?FFV1_V1].{V3}
+- Version 1 of FFV1 adds support of more video bit depths and was flagged as stable on April 24, 2009 [@?FFV1_V1].{V3}
 
-- Version 2 of FFV1 only existed in experimental form and is not described by this document, but is available as a LyX file at <https://github.com/FFmpeg/FFV1/blob/8ad772b6d61c3dd8b0171979a2cd9f11924d5532/ffv1.lyx>.{V3}
+- Version 2 of FFV1 only existed in experimental form and is not described by this document, but it is available as a LyX file at <https://github.com/FFmpeg/FFV1/blob/8ad772b6d61c3dd8b0171979a2cd9f11924d5532/ffv1.lyx>.{V3}
 
-- Version 3 of FFV1 adds several features such as increased description of the characteristics of the encoding images and embedded CRC data to support fixity verification of the encoding. Version 3 has been flagged as stable on August 17, 2013 [@?FFV1_V3].{V3}
+- Version 3 of FFV1 adds several features such as increased description of the characteristics of the encoding images and embedded Cyclic Redundancy Check (CRC) data to support fixity verification of the encoding. Version 3 was flagged as stable on August 17, 2013 [@?FFV1_V3].{V3}
 
 This document defines a version 4 of FFV1. Prior versions of FFV1 are defined within [@?I-D.ietf-cellar-ffv1].{V4}
 
-This document assumes familiarity with mathematical and coding concepts such as Range coding [@?range-coding] and YCbCr color spaces [@?YCbCr].
+This document assumes familiarity with mathematical and coding concepts such as Range encoding [@?Range-Encoding] and YCbCr color spaces [@?YCbCr].
 
-This specification describes the valid bitstream and how to decode such valid bitstream. Bitstreams not conforming to this specification or how they are handled is outside this specification. A decoder could reject every invalid bitstream or attempt to perform error concealment or re-download or use a redundant copy of the invalid part or any other action it deems appropriate.
+This specification describes the valid bitstream and how to decode it.  Nonconformant bitstreams and the nonconformant handling of bitstreams are outside this specification.  A decoder can perform any action that it deems appropriate for an invalid bitstream: reject the bitstream, attempt to perform error concealment, or re-download or use a redundant copy of the invalid part.
```

Comment:
nit: there are double-spaces in between the sentences

Reply:
verified that the double spaces are before and after AUTH48